_rest_routes.inc.php

```diff
@@ -6234,7 +6237,247 @@
         RestConfig::apiLog($return, $data);
         return $return;
     },
+    /**
+     * Schema for the message request
```

Comment:
should be `Schema for the transaction request`

Reply:
resolved